content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-vm.md

```diff
@@ -0,0 +1,852 @@
+---
+title: "VmWare Installation"
+keywords: 'kubernetes, kubesphere, VmWare, installation'
+description: 'How to install KubeSphere on VmWare Linux machines'
+
+
+weight: 2240
+---
+
+# 1、Prepare Environment
```

Comment:
Remove all sequence, please reference this [doc template](https://v3-0.docs.kubesphere.io/docs/installing-on-linux/public-cloud/kubesphere-on-qingcloud-instance/).